pkg/xray/resource_xray_license_policy.go

```diff
@@ -0,0 +1,168 @@
+package xray
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func resourceXrayLicensePolicyV2() *schema.Resource {
+	return &schema.Resource{
+		SchemaVersion: 1,
+		Create:        resourceXrayPolicyCreate,
+		Read:          resourceXrayPolicyRead,
+		Update:        resourceXrayPolicyUpdate,
+		Delete:        resourceXrayPolicyDelete,
+		Description: "Creates an xray policy using V2 of the underlying APIs. Please note: " +
+			"It's only compatible with Bearer token auth method (Identity and Access => Access Tokens",
+
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+
+		Schema: map[string]*schema.Schema{
```

Comment:
You know, this is all heavily nested and hard to read. Maybe the thing to do is extract each sub-map as a variable, and thus visually flatten then, then in this function, stitch them together.

Reply:
fixed